spec/controllers/taxons_controller_spec.rb

```diff
@@ -121,6 +121,9 @@
 
   describe '#confirm_bulk_publish' do
     it 'renders confirm bulk publish' do
+      stub_request(:get, "https://email-alert-api.test.gov.uk/subscriber-lists")
```

Comment:
Since we do this many times, I wonder if could refactor this into a `stub_empty_subscriber_lists` method and then just call that everywhere?